data/domain.yml

```
@@ -123,11 +123,11 @@ responses:
           payload: /intent_insights_advisor_recommendation_by_type
   utter_core_rephrase:
     - text: "I'm sorry. I didn't understand that. Could you try to rephrase your question?"
-    - text: "Sorry I didn't get that. Can you rephrase?"
-    - text: 'I don''t understand. Say "I need help" for information on what I can do.'
+    - text: "Sorry, I didn't get that. Can you rephrase?"
+    - text: 'I don't understand. Say "I need help" for information on what I can do.'
```

```suggestion
    - text: 'I don''t understand. Say "I need help" for information on what I can do.'
```

That extra `'` is required in this case, needs to escape the other one.